server/src/main/java/io/druid/server/http/DatasourcesResource.java

```diff
@@ -200,6 +201,53 @@ public Response deleteDataSource(
     return Response.ok().build();
   }
 
+  /*
+  Uncomment this method once the method deleteSourceName (just above this) is deleted.
```

Comment:
@fjy @nishantmonu51 @rasahner Is it fine to keep commented code? My thought process is that when we remove the deprecated API, we need not to spend time again writing the code.